system-addon/content-src/lib/section-menu-options.js

```diff
@@ -0,0 +1,46 @@
+import {actionCreators as ac, actionTypes as at} from "common/Actions.jsm";
+
+/**
+ * List of functions that return items that can be included as menu options in a
+ * SectionMenu. All functions take the section as the only parameter.
+ */
+export const SectionMenuOptions = {
+  Separator: () => ({type: "separator"}),
+  RemoveSection: section => ({
+    id: "menu_action_remove_section",
+    icon: "dismiss",
+    action: ac.SetPref(section.showPrefName, false),
+    userEvent: "SECTION_MENU_REMOVE"
+  }),
+  CollapseSection: section => ({
+    id: "menu_action_collapse_section",
+    icon: "collapse",
+    action: ac.SetPref(section.collapsePrefName, true),
+    userEvent: "SECTION_MENU_COLLAPSE"
+  }),
+  ExpandSection: section => ({
+    id: "menu_action_expand_section",
+    icon: "collapse",
+    action: ac.SetPref(section.collapsePrefName, false),
+    userEvent: "SECTION_MENU_EXPAND"
+  }),
+  ManageSection: section => ({
+    id: "menu_action_manage_section",
+    icon: "settings",
+    action: {type: at.SETTINGS_OPEN},
+    userEvent: "SECTION_MENU_MANAGE"
+  }),
+  AddTopSite: section => ({
+    id: "menu_action_add_topsite",
+    icon: "add",
+    action: {type: at.TOP_SITES_EDIT, data: {index: -1}},
+    userEvent: "SECTION_MENU_ADD_TOPSITE"
+  }),
+  AboutPocket: section => ({
+    id: "menu_action_about_pocket",
+    icon: "about",
+    redirectUrl: "https://getpocket.cdn.mozilla.net/firefox/new_tab_learn_more",
```

Comment:
We should pass in the `disclaimer_link` as part of the section to use here.

Reply:
Could we change this to AboutSection instead? Section could hold an object that further describes this i.e. section.about? SectionManager can provide this about object based on topstories.options.

Reply:
This just in from the design meeting. This should be "Privacy Notice" and link to https://www.mozilla.org/en-US/privacy/firefox/#suggest-relevant-content which is `info_link` but needs the `#anchor` updated.

Reply:
cool. So we'll have a `PrivacyNotice` option and no `About*` section and just conditionally show it based on if it has the url in the section options.